arelle/plugin/validate/ESEF/__init__.py

```diff
@@ -292,9 +292,15 @@ def checkFootnote(elt, text):
             if doc.type in (ModelDocument.Type.INLINEXBRL, ModelDocument.Type.UnknownXML):
                 _baseName, _baseExt = os.path.splitext(doc.basename)
                 if _baseExt not in (".xhtml",".html"):
-                    modelXbrl.error("ESEF.2.6.1.incorrectFileExtension",
-                        _("Inline XBRL document included within a ESEF report package MUST have a .html or .xhtml extension: %(fileName)s"),
-                        modelObject=doc, fileName=doc.basename)
+                    if val.consolidated:
+                        XHTMLExtensionGuidance = "2.6.1"
+                        reportType = _("Inline XBRL document included within a ESEF report package")
+                    else:
+                        XHTMLExtensionGuidance = "4.1.1"
+                        reportType = _("Stand-alone XHTML document")
+                    modelXbrl.error("ESEF.%s.incorrectFileExtension" % XHTMLExtensionGuidance,
```

Comment:
> I'm not absolutely sure about the severity level here, I suppose it makes sense to use the same one as for iXBRL reports.

2.6.1 says "MUST" and 4.1.1 says "recommends" (as you say), so this seems reasonable to me for now.  We can always change it later if someone disagrees in the future :-).